package.json

```diff
@@ -23,6 +23,7 @@
         "gmail firefox extension"
     ],
     "dependencies": {
+        "gmail-js": "^0.9.9",
```

Comment:
Are we… a dependency on ourself? That sounds weird 😃

Reply:
I've included the old library for testing purposes. Should have been part of the `devDependencies` group I guess.

Reply:
Yeah devDependency makes much more sense. Nice test too!